### ISSUE TITLE
If article doesn't have featured image, it should pull in first img in article.

### ISSUE THREAD

**Issue by isabisa** [unitymakesus/ednc-2020#194]
Nancy said this one isn't working because the first img is too small. We just need to force it.
<img width="1390" alt="Screen Shot 2020-07-09 at 9 35 39 AM" src="https://user-images.githubusercontent.com/928176/87046762-91fab200-c1c7-11ea-995a-c5e56a064b15.png">

**Comment by isabisa** [comment]
Previously told people the images needed to at least 2400px wide so they could be cropped properly. But that they also needed to be optimized before upload.

**Comment by isabisa** [comment]
In editor: Settings for featured image aren't showing up right under it. It needs to be higher on the sidebar.

**Comment by andrewmarino** [comment]
This looks like an issue with the improvements I made to the previous theme's "catch_that_image" function.

https://github.com/unitymakesus/ednc-2020/blob/dev/wp-content/themes/ednc-2020/app/media.php#L64

The problem I tried to solve is the old scrape would return a full src image, sometimes resulting in an image several megabytes large. Returning an attachment instead of an image src will fixes that. But it looks like it doesn't work if a smaller thumbnail size on the attachment was used in the_content?

HMMMM.
